### ISSUE TITLE
Add alternate logo functionality

### ISSUE THREAD

**Issue by obasekietinosa** [PaystackHQ/nigerialogos#160]
Allow multiple logo versions for organizations. 

Example use case would the square versions or dark color versions of the logos.

Proposed implementation would be to add a new field to [logos.json](https://github.com/PaystackHQ/nigerialogos/blob/master/logos.json) entries, say `alternates` and then have the app render a "click here to see alternate versions" below the image that brings up a modal with alternates to download.

**Comment by prezine** [comment]
This makes sense, 
first @Odusanya Can you approve the pending pull request of logos

**Comment by solamichealolawale** [comment]
I agree with @obasekietinosa advise, but I think a carousel or slider of logos would be appropriate with the same functionality on each of the images the way it is.
